media-sound/teamspeak-server-bin/teamspeak-server-bin-3.1.1.ebuild

```diff
@@ -0,0 +1,134 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils multilib systemd user
+
+DESCRIPTION="TeamSpeak is software for quality voice communication via the Internet"
+HOMEPAGE="https://www.teamspeak.com/"
+SRC_URI="amd64? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_amd64-${PV}.tar.bz2 )
+	x86? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_x86-${PV}.tar.bz2 )"
+
+LICENSE="teamspeak3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc mysql tsdns"
+
+RDEPEND="mysql? ( dev-db/mariadb-connector-c )"
+
+RESTRICT="fetch mirror strip"
+
+S="${WORKDIR}/teamspeak3-server_linux_${ARCH}"
+
+pkg_nofetch() {
+	elog "Please download ${A} from ${HOMEPAGE}"
+	elog "and place it in your DISTDIR directory."
+}
+
+pkg_setup() {
+	enewuser teamspeak
+}
+
+src_install() {
+	local TS3_DIR="teamspeak3-server"
```

Comment:
Please don't. Those are single words, so you're saving little and the code below is much harder to read when you have to constantly substitute and verify there's no mixup.

Reply:
Okay, as you wan't. I personally don't see any difference in being reading harder.